### PR TITLE
Starter Kit Items

### PR DESCRIPTION
**Pull request by ravingrabbits32**
Added starter kit items (Mystic Apprentice Top Hat, Dragon Flame Sword, etc.)
Added Mushroom Hollow arena chest loot

### CODE REVIEW

File: Web.AssetBundles/Assets/LocalAssets/InternalLoot.xml [hunk 1]
```diff
@@ -411,6 +411,17 @@
                 <Item itemName="COL_FOOD_Apple01" count="1" weight="2" />
             </Items>
         </LootInfo>
+
```
Comment (FeroxFoxxo) [inline Web.AssetBundles/Assets/LocalAssets/InternalLoot.xml:414]: Remove stray new line character

**Review by FeroxFoxxo — CHANGES_REQUESTED** [review]
LGTM

File: Web.AssetBundles/Assets/LocalAssets/InternalLoot.xml (hunk 1 above)
Comment (ravingrabbits32) [reply Web.AssetBundles/Assets/LocalAssets/InternalLoot.xml]: fixed

**Review by FeroxFoxxo — APPROVED** [review]
LGTM

**Pull request merged**